plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -45,36 +65,42 @@ module.exports = ({containerConfig, appConfig}) => {
         <ConfigurationMapLabel>Container Image</ConfigurationMapLabel>
         <ConfigurationMapValueWithDefault
           value={findNestedPropertyInObject(appConfig, 'image.id')} />
+        {editLink}
       </ConfigurationMapRow>
       <ConfigurationMapRow>
         <ConfigurationMapLabel>Force pull on launch</ConfigurationMapLabel>
         <ConfigurationMapBooleanValue
           value={findNestedPropertyInObject(appConfig, 'image.forcePull')} />
+        {editLink}
```

Comment:
Ps. Is it OK to pass down the same React component instance in multiple places? Shouldn't this be cloned or instantiated in every place? (I could be wrong, just wanted to check)